src/app/main.c

```diff
@@ -38,46 +38,58 @@ static RpmOstreeCommand commands[] = {
 #ifdef HAVE_COMPOSE_TOOLING
   { "compose", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD |
                RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
-    rpmostree_builtin_compose },
+    NULL, rpmostree_builtin_compose },
 #endif
   { "cleanup", 0,
+    "Clear cached/pending data",
     rpmostree_builtin_cleanup },
   { "db", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD,
-    rpmostree_builtin_db },
+    NULL, rpmostree_builtin_db },
   { "deploy", RPM_OSTREE_BUILTIN_FLAG_SUPPORTS_PKG_INSTALLS,
+    "Deploy a specific commit",
     rpmostree_builtin_deploy },
   { "rebase", RPM_OSTREE_BUILTIN_FLAG_SUPPORTS_PKG_INSTALLS,
+    "Switch to a different tree",
     rpmostree_builtin_rebase },
   { "rollback", 0,
+    "Revert to the previously booted tree",
     rpmostree_builtin_rollback },
   { "status", 0,
+    "Get the version of the booted system",
     rpmostree_builtin_status },
   { "upgrade", RPM_OSTREE_BUILTIN_FLAG_SUPPORTS_PKG_INSTALLS,
+    "Perform a system upgrade",
     rpmostree_builtin_upgrade },
   { "reload", 0,
+    "Reload configuration",
     rpmostree_builtin_reload },
   { "initramfs", 0,
+    "Enable or disable local initramfs regeneration",
     rpmostree_builtin_initramfs },
   { "install", 0,
+    "Download and install layered RPM packages",
     rpmostree_builtin_install },
   { "uninstall", 0,
+    "Remove one or more overlay packages",
     rpmostree_builtin_uninstall },
   /* Legacy aliases */
   { "pkg-add", RPM_OSTREE_BUILTIN_FLAG_HIDDEN,
+    "Download and install layered RPM packages",
     rpmostree_builtin_install },
   { "pkg-remove", RPM_OSTREE_BUILTIN_FLAG_HIDDEN,
+    "Remove one or more overlay packages",
     rpmostree_builtin_uninstall },
   { "rpm", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD |
            RPM_OSTREE_BUILTIN_FLAG_HIDDEN,
-    rpmostree_builtin_db },
+    NULL, rpmostree_builtin_db },
```

Comment:
This is an alias for `db`, like `pkg-add` and `pkg-remove`. Maybe let's leave their descriptions `NULL` rather than duplicating? We're never going to print these strings anyway.